services/QuillLMS/client/app/bundles/Teacher/components/shared/articleSpotlight.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import ReactMarkdown from 'react-markdown';
+import { requestGet } from '../../../../modules/request';
+
+interface ArticleSpotlightProps {
+  backgroundColor?: string,
+  blogPostId: string
+}
+
+export const ArticleSpotlight = ({ backgroundColor, blogPostId } : ArticleSpotlightProps ) => {
+  const [blogPost, setBlogPost] = React.useState(null);
+  const [blogPostAuthor, setBlogPostAuthor] = React.useState(null);
+
+  React.useEffect(() => {
+    getBlogPost(blogPostId)
+  }, [])
+
+  function getBlogPost(id: string) {
+    requestGet(`/featured_blog_post/${id}`,
+      (data) => {
+        if(data.blog_post && data.author) {
+          const { blog_post, author } = data;
+          setBlogPost(blog_post)
+          setBlogPostAuthor(author)
+        }
+      }
+    )
+  }
+
+  if(!blogPost) { return <span /> }
+
+  const { title, body, slug } = blogPost;
+  const teacherCenterBaseUrl = `${process.env.DEFAULT_URL}/teacher-center`;
+  const backgroundColorStyle = backgroundColor ? { backgroundColor: backgroundColor } : {};
+
+  return(
+    <div className="article-spotlight-container" style={backgroundColorStyle}>
+      <div className="inner-container">
+        <div className="upper-section">
+          <section className="header-section">
+            <h3>Helpful Article</h3>
+            <p>Want more guidance with your implementation of Quill? Check out these articles written by Quill&apos;s instructional coaches and curriculum team.</p>
+          </section>
+          <a className="quill-button contained primary fun" href={teacherCenterBaseUrl} rel="noopener noreferrer" target="_blank">Show more</a>
```

Comment:
We should try to remember to use the `focus-on-light` or `focus-on-dark` class for interactive elements.